handlers/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: Test and reload apache2
   ansible.builtin.command: apache2ctl configtest
-  changed_when: false
+  changed_when: true
   notify: reload apache2
```

Comment:
I guess this needs to be capitalized. Maybe it was forgotten and in molecule it's never hit.

Reply:
This was not capitalized in many places, but fixed now.